using-timescaledb/distributed-hypertables.md

```diff
@@ -48,32 +54,175 @@ CREATE TABLE conditions (
+partitioning configuration, each data node will be responsible for a
+specific subset of the data based on the value of `location`, allowing
+concurrent and parallel execution of a query across the data nodes.
+
+You can now insert data into the distributed hypertable and the data
+will automatically be routed to the correct data node.
+
+#### Best Practices for Partitioning
+
+Like a regular hypertable, a distributed hypertable needs to be
+partitioned along a "time" dimension (e.g., a `timestamptz`
+column). However, for most distributed workloads, we recommend
+multi-dimensional partitioning with an additional "space" dimension
+that consistently partitions the data over the data nodes, similar to
+traditional
+[sharding](https://en.wikipedia.org/wiki/Shard_(database_architecture)). If
+your data set has a column called something similar to `customerID`,
+`deviceID`, or `location` (as in the example above), and it figures
+frequently in the `GROUP BY` clause of queries, then it is likely a
+good candidate column for space partitioning.
+
+With space partitioning, multi-row inserts can often be split across
+the data nodes for increased insert performance, and queries can be
+parallelized by executing a subset of the query on each data
+node. TimescaleDB contains many optimizations to take advantage of the
+partitioning scheme to make best use of multi-node resources.
+
+While a distributed hypertable works with only time partitioning it
+will likely lead to inserts on only one data node at a time. Assuming
+in-order inserts, only one chunk gets written at a time and each chunk
+is placed on a data node in round-robin fashion. Similarly, queries
+cannot be optimally parallelized unless they cover as many chunk time
+intervals as there are data nodes.
+
+### Inserting into a Distributed Hypertable [](insert)
+
+Just like a regular hypertable, it is important to batch data when
+inserting into a distributed hypertable. Each insert statement is its
+own transaction, and the overhead and cost of this transaction is good
+to amortize over many rows of data. With a distributed hypertable, the
+transaction has additional costs due to the coordination that needs to
+happen across data nodes (e.g., two-phase commit protocol). There are
+two ways to insert data:
+
+- `INSERT`: the access node sets up a multi-row prepared statement on
+  each data node and then splits the original insert statement across
+  these sub-statements. Each prepared statement allows buffering of up
+  to `timescaledb.max_insert_batch_size` number of rows (default 1000)
+  before the statement is complete and thus written to the data
+  node. Throughput can be tuned by setting a different batch size,
+  although the max size is limited by the max number of parameters of
+  a prepared statement (32767), and the number of columns in each row.
+  For example, if a distributed hypertable has 10 columns, the max
+  insert batch size is capped at 3276 rows.
```

Comment:
is this the max insert batch size _per data node_ or overall max?

Reply:
Per data node. I'll clarify.